content/millennium/r4/other/health-cards.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Health Cards | R4 API
+---
+
+# Health-Cards
+
+* TOC
+{:toc}
+
+## Overview
+
+A Health Card is a verifiable, digital artifact that represents a limited set of information about an individual, relevant to a specific health status. The card can be presented in digital and physical form. A QR code representing the card can be shared from a device or a piece of paper. Information in the health card can be used to determine the authenticity of the artifact and its issuer. The goal is to provide individuals with the choice to share limited health information in a portable and secure manner.
+
+Health cards are designed in conformance with the emerging [SMART Health Card framework](https://smarthealth.cards) and [HL7 FHIR Vaccination and Testing IG](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/).
```

Comment:
The URL for the spec is now: https://spec.smarthealth.cards/

The smarthealth.cards URL now directs to a consumer-facing FAQ - we may want to include that too, but the intention here seems to be linking to the technical spec.

Reply:
How about this? https://github.com/cerner/fhir.cerner.com/pull/659/commits/010373b4f33500144ed55e579fc13e86d741d51a#diff-bf1e7917ef250f45c2b8a4a1b46b31dc38eb42ee5dd2957211f0ba40692d42b1R14